doc/syslog.md

```diff
@@ -0,0 +1,207 @@
+## Syslog Support
+
+The system comes with native logging to internal storage, `/var/log/`,
+or `/log` for a shortcut.  Depending on the device, this may be a RAM
+disk, meaning logs are not retained across reboots.  This document
+details how to log to external media or remote syslog servers.
+
+It is also possible to set up the device to act as a syslog server (log
+sink), this is covered briefly at the very end of this document.
+
+> **Note:** the native logging cannot be modified, only the log file
+> rotation can be changed.  Please see the `dir` admin-exec command for
+> a listing of existing native log files.
+
+
+### Log to File
+
+Logging to a local file is useful when combined with an external media.
+E.g., a USB stick with a log partition (named/labeled: "log").  Below is
+an example.
+
+For a list of available log facilities, see the table in a later section.
+
+```bash
+admin@example:/> configure
+admin@example:/config/> edit syslog
+admin@example:/config/syslog/> edit actions log-file file:/media/log/mylog
+admin@example:/config/syslog/actions/log-file/file:file:/media/log/mylog/> set facility-list
```

Comment:
typo file: ?

Reply:
Good question, I'll have to double check.  I remember it looked funny with the `:` in the prompt, but it could very well be copy-paste error.  Thanks!

Reply:
You're right:

```
admin@foo:/config/> edit syslog 
admin@foo:/config/syslog/> edit actions log-file file:/tmp/foo.log
admin@foo:/config/syslog/actions/log-file/file:/tmp/foo.log/> 
                                 ~~~~~~~~~~~~~~
```